src/dicom-parse.c

```diff
@@ -954,16 +954,11 @@ bool dcm_parse_pixeldata_offsets(DcmError **error,
         // the BOT is missing, we must scan pixeldata to find the position of
         // each frame
 
-        // we could use our generic parser above ^^ but we have a special loop
-        // here as an optimisation (we can skip over the pixel data itself)
```

Comment:
I like this comment, it explains why we need to do our own parsing here and not just rely on the generic parser above.